static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -137,9 +143,14 @@ type State = AsyncView['state'] & {
     [key: string]: string[];
   };
   environments: Environment[] | null;
+  issueCount: number;
+  loadingPreview: boolean;
+  memberList: IndexedMembersByProject | null;
+  previewCursor: string | null | undefined;
```

Comment:
can we move all this to the preview component now that its no longer triggered by the parent?

Reply:
what do you mean by this? I thought these are still triggered by the condition form

Reply:
mostly there's now quite a bit of state for this component, it could be all contained within the preview

Reply:
Not quite sure how it can be contained within the preview, these 4 states all rely on any changes made in the condition form